frontend/src/app/modules/new-meeting/new-meeting/new-meeting.component.ts

```diff
@@ -30,9 +30,12 @@ export class NewMeetingComponent extends BaseComponent implements OnInit {
 
     public durations: IDuration[] = getDisplayDuration();
 
-    public locations = Object.keys(LocationType).filter(key => Number.isNaN(Number(key)));
+    public locations: string[] = [
+        LocationType[LocationType.Zoom],
+        LocationType[LocationType.GoogleMeet],
+        LocationType[LocationType.Office]];
 
-    public unitOfTime = Object.keys(UnitOfTime).filter(key => Number.isNaN(Number(key)));
+    public unitOfTime: string[] = [UnitOfTime[UnitOfTime.min], UnitOfTime[UnitOfTime.hour]];
```

Comment:
it would be better to change  enums or use something like keyof typeof {{enum name}}

Reply:
But I don't have this code in file now. I refactored it to using labelMapper.